examples/testing/datasets/mnist/3.0.1/dataset_info.json

```diff
@@ -0,0 +1,105 @@
+{
+  "citation": "@article{lecun2010mnist,\n  title={MNIST handwritten digit database},\n  author={LeCun, Yann and Cortes, Corinna and Burges, CJ},\n  journal={ATT Labs [Online]. Available: http://yann. lecun. com/exdb/mnist},\n  volume={2},\n  year={2010}\n}\n",
```

Comment:
We should add a README to examples/testing/datasets explaining when and how to add entries here.

Reply:
@mohitreddy1996 tells me that this already exists in #287